lib/elixir/lib/calendar/datetime.ex

```diff
@@ -1674,6 +1675,128 @@ defmodule DateTime do
     end
   end
 
+  @doc """
+  Shifts given `datetime` by `duration` according to its calendar.
+
+  Allowed units are: `:year`, `:month`, `:week`, `:day`, `:hour`, `:minute`, `:second`, `:microsecond`.
+
+  Shifting datetimes in time zones that observe "Daylight Saving Time" across
+  summer/winter time will add/remove one hour from the resulting datetime.
+  This ensures `shift/3` always returns a valid datetime.
```

Comment:
I think this could warrant an example illustrating it, WDYT?

Reply:
done in https://github.com/elixir-lang/elixir/pull/13385/commits/5e9d6fbe5f25e3c5a263f825aa7266c5190f2742